tdvt/tdvt/exprtests/pretest/connection_tests/staples/setup.staples_connection_test.xml

```diff
@@ -0,0 +1,220 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<logical-query>
+  <selects>
+    <field>[PID]</field>
+    <field>[sum:Gross Profit:ok]</field>
+    <field>[sum:Sales Total:ok]</field>
+  </selects>
+  <domainOp class="logical-operator" type="snowflake">
```

Comment:
So this is saying a good smoke test for the Staples table being loaded correctly is roughly (forgive my rusty SQL):

SELECT 'PID', SUM('Gross Profit'), SUM('Sales Total')
FROM 'Staples'
WHERE 'Call Center Region' == EAST && 'Order Date' == 2002
GROUP BY 'PID'
HAVING ('Sales Total' > -3640.23 && 'Sales Total' < 24622.400000000001) 

Did we pull this from somewhere else or an evaluation you did of what data from Staples is best to check?

Before this change what was checked?

Reply:
Honestly I just copied a similar Staples logical test to use as a smoke test.  It was just to validate that I could get a logical smoke test to run.  If there is a better test I'm all for it but it looks like it's rarely used in the connector ini files.  

What I checked before hand was that I could successfully run the smoke tests against postgres before the change, then after made sure it could load the logical tests in that format.